runtime/frequency/src/lib.rs

```diff
@@ -146,17 +149,205 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
+	remove_sudo::RemoveSudo,
 >;
 
+// ==============================================
```

Comment:
I think it would be helpful if we can separate migration logic into it's own files and have a repo or document of all previous migrations in future.

Reply:
That was a problem to do in this case, because there was kind of a circular dependency on runtime, in part due to the sudo pallet being removed. Parity actually recommended that this migration go directly in here.  For other pallet storage migrations it should be done that way.

Reply:
I think it would be a great idea to somehow store old migrations somewhere. Open to more specific ideas.